server/policies/import.ts

```diff
@@ -0,0 +1,16 @@
+import { ImportState } from "@shared/types";
+import { User, Team, Import } from "@server/models";
+import { allow } from "./cancan";
+import { and, isTeamAdmin, isTeamMutable } from "./utils";
+
+allow(User, ["createImport", "readImport"], Team, (actor, team) =>
```

Comment:
```suggestion
allow(User, ["createImport", "listImports"], Team, (actor, team) =>
```

This would be more typical